### DISCUSSION TITLE
FN + Q shortcut interfering with Game

### DISCUSSION THREAD

**Discussion by a1exsDev** [Open-Shell/Open-Shell-Menu#1357] [Q&A]
I am playing a game requiring me to press FN + Q to show some tips for the game and when I run the shortcut, it opens the File Explorer instead. This did not happen before installing Open Shell and I cannot figure out how to disable it. Is there any way I can fix this issue without uninstalling Open Shell? (The Game does not allow me to change the keybinding for that specific shortcut and I know it is Open Shell because It opens File Explorer on My PC which is set to open there in the Open Shell Settings.)  - A Windows User

**Comment by Imold** [comment]
FN isn't a normal key, it's a hardware level modifier key, it doesn't send a key code and can't be detected by software. FN functions have no standardization so you'll have to tell us what the function of FN + Q is. What is it the game is actually asking you to press? because no program would ask you to press FN + AnyKey because FN can only be seen by hardware.
